### DISCUSSION TITLE
Taro 从3.2.0升级到3.5.7程序run起来报错

### DISCUSSION THREAD

**Discussion by POI-ShiFeng** [NervJS/taro#12790] [Q & A]
### 这个特性解决了什么问题？
为了解决编译速度与包大小问题

### 这个 API 长什么样？
3.5.7升级

### 以下为升级后报错信息

The above error occurred in the <Provider> component:

    at Provider (http://127.0.0.1:26235/appservice/prebundle/vendors-node_modules_taro_weapp_prebundle_react-redux_js.js?t=wechat&s=1667822309603&v=611b4540db4f695a82cc1aaa903be43e:1214:22)
    at App (http://127.0.0.1:26235/appservice/app.js:175:166)
    at AppWrapper (http://127.0.0.1:26235/appservice/prebundle/vendors-node_modules_taro_weapp_prebundle_tarojs_plugin-framework-react_dist_runtime_js.js?t=wechat&s=1667822309603&v=da9b3d84db930a5354d86bad5c9eb200:308:13)

Consider adding an error boundary to your tree to customize error handling behavior.
app.js错误:
 Error: Invalid hook call. Hooks can only be called inside of the body of a function component. This could happen for one of the following reasons:
1. You might have mismatching versions of React and the renderer (such as React DOM)
2. You might be breaking the Rules of Hooks
3. You might have more than one copy of React in the same app
See https://reactjs.org/link/invalid-hook-call for tips about how to debug and fix this problem

**Comment by yoyo837** [comment]
看看是不是有依赖未对齐, 然后删除node_modules和lock文件全新安装试一试

**Comment by POI-ShiFeng** [comment]
> 看看是不是有依赖未对齐, 然后删除node_modules和lock文件全新安装试一试

编译没有问题，是不是3.5的语法哪里做了调整

**Comment by TheKonka** [comment]
hook调用位置错了，给个能复现的demo 别人也能直接帮你定位问题

**Comment by POI-ShiFeng** [comment]
> hook调用位置错了，给个能复现的demo 别人也能直接帮你定位问题

我这里报错的位置是在node_modules里面，不在本身的业务代码里

**Comment by yoyo837** [comment]
提供复现和报错位置在不在node_modules没关系

**Comment by POI-ShiFeng** [comment]
[https://bfe.oss-cn-hangzhou.aliyuncs.com/taro-3.5.7-error.png](url)
凭这张图，看得出问题不

> 提供复现和报错位置在不在node_modules没关系

**Comment by ZakaryCode** [comment]
1. 请不要将提问在 issue 中呈现并以 enhancement 的形式提交，该 issue 将转到 discussions 中继续
2. 问题中缺失可复现的 demo，缺失对应编译平台等重要信息
3. 通过仅有的报错只能判断与 react 或相关库、语法等有关联，可关闭 prebundle 特性或通过排除对应以来自行排查

**Comment by Ren-zhch** [comment]
这个最后怎么解决的
